run-tests-with-build-optawebs-parent/optawebs-project-sources-test/pom.xml

```diff
@@ -12,6 +12,14 @@
 
     <artifactId>optawebs-project-sources-test</artifactId>
 
+    <properties>
+        <sources.directory>${sources.directory.module.project.sources.dependency.get}</sources.directory>
```

Comment:
I think this is done already in the parent in profile `optawebs-project-sources`, so the property should be set for this module already when invoking using `-Dtest.type=project-sources`, so here it might be not necessary.

Reply:
Fixed